### ISSUE TITLE
ImportError: No module named 'gprMax.fields_updates'

### ISSUE THREAD

**Issue by zhj-insomnia** [gprMax/gprMax#123]
Dear developers
Thank you for developing and sharing the gprMax for free，it's a great tool, and I'm really interest in it.
However, after I installed it as requested, I found that the following error occurred when I entered the *.in file. The *.in file named a.in in C:/ .
_(gprMax) c:\Windows\System32\gprMax>python -m gprMax C:/a.in
ImportError: No module named 'gprMax.fields_updates'_ 

I'm using the windows10-64bit. 
This is the first time I have used this software, and I really don't know what's wrong with it.
I would appreciate it if you could give me a reply during your busy schedule.
Best wishes

**Comment by craig-warren** [comment]
Did you successfully complete the `python setup.py build` and `python setup.py install` steps? Can you try with the example model `cylinder_Ascan_2D.in` please.

Kind regards,

Craig

**Comment by zhj-insomnia** [comment]
Thank you for your reply.
I did it last night.
I don't konw why did it works, but there are two conjectures.
First, I installed it in C:/Windows/System32/, I guess that due to the administrator privileges, something can not write in this folder.
Second, maybe it was because of the network speed. I am in china.
So, I reinstalled it in C:/. I am pretty confirm that python setup.py build and python setup.py install completed.
Thank you for offering such a great software.
